config.json

```diff
@@ -25,87 +25,6 @@
       }
     ],
     "artifacts": [
-      {
-        "groupId": "com.github.bumptech.glide",
```

Comment:
What's with this change?  It looks like this was (nominally?) sorted by (groupId, artifactId), but this entire section for `com.github.bumptech.glide` was moved to *after* `com.google.mlkit`.  *Is* this data sorted in any way?

Reply:
no. It is not sorted

I did sort it in large PR and must sort it, because it is a huge. It even needs cleanup (duplicate and unneeded `dependencyOnly= "true"`with old versions).
I need to do it in separate PR after all versions are up-to-date.

I even started adding comments in json, but never finished it.

Reply:
> Additionally, I don't know if the script will preserve the comments, since comments are not part of the JSON standard.

I didn't test update script, but binderator and other tools (serialization - Newtonsoft and System.Json) didn't have issues with comments and I was tweaking versions a lot.

Reply:
Sorting is nice, but overkill in this case, because 90%+ of the work with `config.json` is manual (adding new packages, tweaking versions, cleanup of duplicate entries or old/stale packages).